command/views/hook_ui.go

```diff
@@ -69,9 +69,6 @@ const (
 
 func (h *UiHook) PreApply(addr addrs.AbsResourceInstance, gen states.Generation, action plans.Action, priorState, plannedNewState cty.Value) (terraform.HookAction, error) {
 	dispAddr := addr.String()
-	if gen != states.CurrentGen {
-		dispAddr = fmt.Sprintf("%s (%s)", dispAddr, gen)
-	}
```

Comment:
While this code currently does nothing, I think this is probably a bug in a call site somewhere else, so I'm going to undo this change and add coverage. This will allow us to investigate and possibly fix the bug in a later PR.